config/faker-generator.js

```diff
@@ -0,0 +1,18 @@
+const _ = require('underscore');
+const fs = require('fs');
```

Comment:
I think what this would look like with the `unit.js` example is:
```js
const req = require.context('../test/fixtures/config/', true, /^(.*\.(js$))[^.]*$/i);

req.keys().forEach(key => {
  const entityFactory = req(key);
  const dest = key.replace('/config'. '/collections').replace('.js', '.json');

  // etc
});
```
That said, this is node so what you have is fine

Reply:
so the first thing I ran into was `context` not being a function...because I was running this through node instead of webpack? Is that a thing?

Reply:
ah yeah I dunno maybe that's the case.  FWIW, if it's moved to a cypress plugin I believe that'd no longer be a problem anyhow.  It's kind of interesting as a Cypress plugin.. might be worth publishing at some point

Reply:
Yeah, when I first started on this I was thinking that with a little work it could be pushed up separately.